### PR TITLE
Use proper naming convention of type 6

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies
See discussion here https://github.com/thoth-station/storages/pull/2134#discussion_r535906381

## This introduces a breaking change

- [X] Yes
- [ ] No

All the applications that are using this method should reference it with its new name

## This should yield a new module release

- [ ] Yes
- [X] No

## This Pull Request implements
Rename `get_kebechet_github_installations_count_per_is_active` to `get_active_kebechet_github_installations_repos_count_all` - the use of proper naming convention

## Description
 @pacospace wrote
> naming convention for get_kebechet_github_installations_count_per_is_active is not respected actually. Whatever is returning type number should be should have _count at the end => get_active_kebechet_github_installations_repos_count_all (Type six naming)

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -3475,7 +3475,7 @@ def update_kebechet_github_installations_on_is_active(self, slug: str) -> bool:
                 return True
             return False
 
-    def get_kebechet_github_installations_count_per_is_active(self) -> int:
+    def get_active_kebechet_github_installations_repos_count_all(self) -> int:
```
Comment (pacospace) [inline thoth/storages/graph/postgres.py:3478]: Please add also example output here :)

**Review by pacospace — COMMENTED** [review]
Thanks @xtuchyna !! Please check with @saisankargochhayat , which components are affected by these changes, because after new release we need to have those adjusted already :)

**Comment by saisankargochhayat** [comment]
> Thanks @xtuchyna !! Please check with @saisankargochhayat , which components are affected by these changes, because after new release we need to have those adjusted already :)

This is the only one I can think of - https://github.com/thoth-station/metrics-exporter/blob/a4cdd15bd45896a51ae167c42f09487f00b0b45b/thoth/metrics_exporter/jobs/kebechet.py#L39

**Comment by pacospace** [comment]
> > Thanks @xtuchyna !! Please check with @saisankargochhayat , which components are affected by these changes, because after new release we need to have those adjusted already :)
> 
> This is the only one I can think of - https://github.com/thoth-station/metrics-exporter/blob/a4cdd15bd45896a51ae167c42f09487f00b0b45b/thoth/metrics_exporter/jobs/kebechet.py#L39

Thanks @saisankargochhayat :) :rocket:

**Review by pacospace — APPROVED** [review]

**Pull request merged**